app/helpers/orchestration/orchestrator.rb

```diff
@@ -0,0 +1,168 @@
+# frozen_string_literal: true
+
+# Orchestrator: Methods for managing active playbook context
+module Orchestration::Orchestrator # rubocop:todo Metrics/ModuleLength
+  include Orchestration::Playbooks
+
+  available = Orchestration::Playbooks.constants.reject { |m| m == :Templates }
+  modules = available.map { |m| ('Orchestration::Playbooks::' + m.to_s).constantize }
+
+  playbooks = {}
+  modules.each { |m| playbooks[m::NAME] = m::PLAYBOOK }
+
+  PLAYBOOKS = playbooks.freeze
+
+  def workflow_configuration(playbook, workflow, option)
+    # For now, assume that if no workflow (ie it is nil, it is a general option)
+    base_configuration = workflow.nil? ? PLAYBOOKS.dig(playbook, :general, :base, option) : PLAYBOOKS.dig(playbook, :workflows, workflow, :base, option)
```

Comment:
```suggestion
    base_configuration = PLAYBOOKS.dig(playbook, workflow || :general, :base, option)
```

Reply:
Admittedly a Ruby noob, but I don't understand why eliminating the :workflows symbol is desirable. Doesn't that key direct where to start looking for the workflow value?

Reply:
Sorry, I misread the second part of the line as `PLAYBOOKS.dig(playbook, workflow, :base, option)` thinking we could simplify it. Will resolve this!